### DISCUSSION TITLE
Geräte erneut an Zentrale anlernen ohne Erfolg nach Update auf 3.65.11

### DISCUSSION THREAD

**Discussion by amvseven** [OpenCCU/OpenCCU#2083] [🇩🇪 Fragen&Antworten]
Hallo, ich bin neu hier, herzliche Grüße an die Community!

Ich möchte homematic Geräte neu direkt an CCU Zentrale (ELV Charly) nach einem Update anlernen. Das Update auf 3.65.11 war erfolgreich. Der Anlernmodus in der WebUI kann gestartet werden. Aber Anlernversuche von verschiedenen Geräten führen zu keinem Posteingang. Versucht habe ich HM-Dis-EP-WM55, HMIP-PS und HMIP-STHO. Ich habe auch jeweils die Rückstellung auf Werkseinstellung vorgenommen.

Allerdings habe ich bei HM-Dis-EP-WM55 anhand der Blinkfolge einen Hinweis in der Anleitung, dass Zurücksetzen nicht möglich ist, aufgrund bereits früherer erfolgreicher Anmeldung. Wie aber kann ich dann überhaupt noch Zugriff auf das Gerät bekommen. Kann jemand meinen Denkfehler aufklären?

Kann eventuell der Systemschlüssel aus der CCU etwas damit zu tun haben? Wird der in den Geräten hinterlegt? Aber sollte ja doch bei Werkseinstellung auch gelöscht werden, oder?

Vielen Dank!

**Comment by MichaelN0815** [comment]
Wieso sollte ein erneutes Anlernen nach einem FW Update notwendig sein? Vielleicht mal die ganze Geschichte erzählen?

**Reply by amvseven** [reply]
Stimmt! Hatte vergessen zu erwähnen, dass ich die CCU neu aufgesetzt habe mit neuer Speicherkarte, ohne Backup.

**Reply by amvseven** [reply]
Update: Ich konnte alle Komponenten bis auf 2 zurücksetzen und wieder anlernen. Ein HMIP-STHO und ein HMIP-WTH-2 lassen sich nicht in der Werkszustand zurückversetzen. Die Led blinkt 1x rot anstatt schnelle Blinkfolge Gelb, wie laut den Anleitungen zu erwarten. Der Systemschlüssel wird offensichtlich in den Komponenten hinterlegt und hatte wohl meinen ersten Versuch vereitelt, da ich in der CCU einen neuen vergab.

Wenn jemand zu den verbliebenen 2 Komponenten eine Idee hat, bitte. Darüber hinaus würde mich interessieren, ob man eine bereits angelernte Komponente ohne sie zurückzusetzen und ohne erneuten Anlernvorgang in die Geräteliste der CCU bekommen kann. Logischerweise nicht über die WebUI, sondern editieren des config-scripts. Also praktisch das was halt ein backup einspielen würde. Wenn jemand eine Quelle dazu hat...
